src/components/Calender.vue

```diff
@@ -174,6 +183,20 @@
 			hsl(0, 100%, 38%, 0.5));
 	}
 
+	@media (min-width: 1400px) {
```

Comment:
If you flip this to `max-width`, you can move the styling to the main block, and move the `display: none` rule to the media query. This way you could re-add the `margin-bottom: auto` to the h3 at the same time.

Alternatively, if you want this to be a `min-width` rule, just remove the auto margin here, and leave it in the initial rule.